client/src/components/DatasetInformation/DatasetDetails.vue

```diff
@@ -124,7 +124,9 @@ onUnmounted(() => {
 
                 <JobDestinationParams v-if="currentUser?.is_admin" :job-id="dataset.creating_job" />
 
-                <JobDependencies v-if="jobDetails.dependencies" :dependencies="jobDetails.dependencies" />
+                <JobDependencies
+                    v-if="!jobLoading && jobDetails.dependencies"
```

Comment:
Could you instead change `jobLoading` to `hasJobDetails` or something like that, meaning this variable can't revert from true to false ?